docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -865,6 +868,25 @@ client.executeQuery(sql1).thenCombine(
     });
 ```
 
+Connect to a {{ site.ccloud }} ksqlDB cluster <a name="connect-to-cloud"></a>
+-----------------------------------------------------------------------------
+
+Use the following code snippet to connect your Java client to a hosted ksqlDB
+cluster in {{ site.ccloud }}.
+
+```java
+ClientOptions options = ClientOptions.create()
+ .setBasicAuthCredentials("<ksqlDB-API-key>", "<ksqlDB-API-secret>")
+ .setHost("<ksqlDB-endpoint>")
+ .setPort(443)
+ .setUseTls(true)
+ .setUseAlpn(true);
+```
+
+Get the API key and endpoint details from your {{ site.ccloud }} cluster.
+For more information, see 
+[Connecting ksqlDB to Confluent Cloud](https://docs.confluent.io/cloud/current/cp-component/ksql-cloud-config.html#connecting-ksqldb-to-ccloud).
```

Comment:
This is not the correct link. This link is for connecting on-prem ksqlDB to a CCloud Kafka cluster. The links you actually want are [this one for creating a CCloud ksqlDB API key](https://docs.confluent.io/cloud/current/cp-component/ksqldb-ccloud-cli.html#create-an-api-key-for-ccloud-ksql-cloud-through-the-ccloud-cli) and [maybe this one since the first bullet shows how users can get the URL of their CCloud ksqlDB cluster](https://docs.confluent.io/cloud/current/cp-component/ksqldb-ccloud-cli.html#access-a-ksql-cloud-application-in-ccloud-with-an-api-key).